go/client/cmd_team_generate_seitan.go

```diff
@@ -0,0 +1,92 @@
+// Copyright 2017 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package client
+
+import (
+	"context"
+	//"errors"
+
+	"github.com/keybase/cli"
+	"github.com/keybase/client/go/libcmdline"
+	"github.com/keybase/client/go/libkb"
+	"github.com/keybase/client/go/protocol/keybase1"
+)
+
+type CmdTeamGenerateSeitan struct {
+	libkb.Contextified
+	Team string
+	Role keybase1.TeamRole
+}
+
+func newCmdTeamGenerateSeitan(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Command {
+	return cli.Command{
+		Name:         "generate-seitan",
+		ArgumentHelp: "<team name>",
+		Usage:        "Generate no-server-trust \"Seitan\" token.",
```

Comment:
`"Generate an invite token that you can send via SMS, iMessage, or other similar mechanism."`